docs/Contributing.md

```diff
@@ -25,44 +25,40 @@ To contribute, please fork the repo and develop in a feature branch off the curr
 ### For non-Windows ###
 1. Run `build.sh`
 
-### Changing build configurations ###
-To build in `DEBUG` mode - set the environment variable `DN3B` to `DEBUG`
-To build in `RELEASE` mode (default) - set the environment variable `DN3B` to `RELEASE`
-
 ## Running ##
 1. Run `dotnet new3` at the command line
```

Comment:
this also seems to be left over from previous approach.
I feel like this section is better explanation: https://github.com/dotnet/templating#how-to-build-run--debug-the-latest. Probably we can reference it here or move it to this article and reference it from readme.md.

Reply:
Removed the section from Readme.md and added into Contributing.md. There are also some minor modifications.